### PR TITLE
Add 'black' to ci

### PR DESCRIPTION
**Pull request by mkst**

### CODE REVIEW

**Comment by simonlindholm** [comment]
Seems you renamed the wrong job, looks okay otherwise. Two small doubts:
- should we pin the black version? Unclear either way, probably won't ever actually matter
- the readme says it's fine to skip black, in the hopes of making the barrier to contributions lower. Is that a policy that should be kept? Maybe keep it but still run black in CI?

**Comment by mkst** [comment]
I've now fixed the job name and pinned black. I don't feel it's a big ask for contributors to run `black` but guess it's fine for the test to fail in CI and contributions still be merged?

**Comment by simonlindholm** [comment]
Can you update the black version in https://github.com/simonlindholm/asm-differ/blob/main/.pre-commit-config.yaml to the same thing?

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ on:
 permissions: read-all
 
 jobs:
-  tests:
+  black:
```
Comment (simonlindholm) [inline .github/workflows/unit-tests.yml:10]:
```suggestion
  tests:
```

**Pull request merged**